test/functional/database-schema/column-types/postgres/column-types-postgres.ts

```diff
@@ -91,6 +91,7 @@ describe("database schema > column types > postgres", () => {
                 post.uuid = "0e37df36-f698-11e6-8dd4-cb9ced3df976"
                 post.json = { id: 1, name: "Post" }
                 post.jsonb = { id: 1, name: "Post" }
+                post.jsonpath = "$.name"
```

Comment:
please add a test checks below like we did for another column types

Reply:
added assertions for jsonpath